diff --git a/node_modules/vrchat/api.ts b/node_modules/vrchat/api.ts
index 4908572..7295951 100644
--- a/node_modules/vrchat/api.ts
+++ b/node_modules/vrchat/api.ts
@@ -7852,7 +7852,7 @@ export const InstancesApiAxiosParamCreator = function (configuration?: Configura
             assertParamExists('sendSelfInvite', 'worldId', worldId)
             // verify required parameter 'instanceId' is not null or undefined
             assertParamExists('sendSelfInvite', 'instanceId', instanceId)
-            const localVarPath = `/instances/{worldId}:{instanceId}/invite`
+            const localVarPath = `/invite/myself/to/{worldId}:{instanceId}`
                 .replace(`{${"worldId"}}`, encodeURIComponent(String(worldId)))
                 .replace(`{${"instanceId"}}`, encodeURIComponent(String(instanceId)));
             // use dummy base URL string because the URL constructor only accepts absolute URLs.
diff --git a/node_modules/vrchat/dist/api.js b/node_modules/vrchat/dist/api.js
index 2e12dd3..e0d881f 100644
--- a/node_modules/vrchat/dist/api.js
+++ b/node_modules/vrchat/dist/api.js
@@ -4337,7 +4337,7 @@ exports.InstancesApiAxiosParamCreator = function (configuration) {
                     common_1.assertParamExists('sendSelfInvite', 'worldId', worldId);
                     // verify required parameter 'instanceId' is not null or undefined
                     common_1.assertParamExists('sendSelfInvite', 'instanceId', instanceId);
-                    localVarPath = "/instances/{worldId}:{instanceId}/invite"
+                    localVarPath = "/invite/myself/to/{worldId}:{instanceId}"
                         .replace("{" + "worldId" + "}", encodeURIComponent(String(worldId)))
                         .replace("{" + "instanceId" + "}", encodeURIComponent(String(instanceId)));
                     localVarUrlObj = new URL(localVarPath, common_1.DUMMY_BASE_URL);
